### DISCUSSION TITLE
How to use unocss to compile pure css in a vite project.

### DISCUSSION THREAD

**Discussion by quank123wip** [unocss/unocss#3747] [Q&A]
I'm working on a small project and trying to use unocss. The project simply contains a ts file and a css file. The expected behavior is that the ts file is bundled, and the css file is compiled into vanilla css with unocss engine. But in the bundle the css isn't compiled. What should I do?

The code is [here](https://github.com/quank123wip/milkdown-theme-baka)

**Comment by domisolo** [comment]
I had the same problem, but everything is OK in windicss. I hope @antfu can support it.

**Reply by userquin** [reply]
add `// @unocss-include`  to your ts file: check https://unocss.dev/guide/extracting#extracting-from-build-tools-pipeline

**Reply by domisolo** [reply]
Thanks, everything is OK when extracting files. I am using UnoCSS in similar React Native Framework and there is no html file in project, so I got this error
![image](https://github.com/unocss/unocss/assets/17431988/0c9d5b50-f103-4aff-a66a-e487723b8a65)

I tried setting unocss option **mode:"per-module"**, but no class was parsed
```jsx
<View className='flex justify-center items-center'>
  <Text>Hello UnoCSS</Text>
</View>
```

<img src="https://github.com/unocss/unocss/assets/17431988/89a68310-e258-4c82-8b6a-4445c986879d" width="35%" />

However, everything is OK in WindiCSS.

<img src="https://github.com/unocss/unocss/assets/17431988/41ea0fe7-c765-4d80-ae18-74b83a419ee7" width="35%" />

Expect your reply.

**Reply by userquin** [reply]
Check this https://github.com/eagermko/react-native-unocss
